.eslintrc.json

```diff
@@ -5,6 +5,7 @@
   "rules": {
     "no-console": "warn",
     "require-atomic-updates": "warn",
+    "lines-between-class-members": ["error", "always", { "exceptAfterSingleLine": true }],
```

Comment:
Why not the default in our config?

Reply:
We need to fix it in individual projects first (especially API).